### DISCUSSION TITLE
6.x mailer markdown loses strikethrough support from 5.x

### DISCUSSION THREAD

**Discussion by brucek2** [laravel/framework#33132] [General]
- Laravel Version: 6.18.18
- PHP Version: 7.4.6
- Database Driver & Version: n/a

### Description:

After upgrading my app from framework 5.7.29 to 6.18.18, I noticed my email templates that included markdown for strike-through no longer rendered as they did before. The strikethrough wrapper ~~ was passed as-is instead of causing strikethrough.

I believe the source of this issue is the switch to commonmark, and the solution would be as easy as including an option to instantiate commonmark using `new GithubFlavoredMarkdownConverter()` in place of  `new CommonMarkConverter()`

### Steps To Reproduce:

1. Create or edit any markdown mail template to include strikethrough text: ~~text~~
2. Render the mail template

**Comment by driesvints** [comment]
It's indeed a feature of Github flavored markdown. But as this is more of a feature request I'm moving this to discussions instead. You can attempt a PR if you want.
